python2/ChocolateyPackageInstaller.ps1

```diff
@@ -0,0 +1,269 @@
+﻿<##################################################################################################

```

Comment:
It looks like you have two artifacts for Python 2, with different folder names. Am I missing something?